client/my-sites/plans/main.jsx

```diff
@@ -36,7 +36,7 @@ const Plans = React.createClass( {
 	getDefaultProps() {
 		return {
 			intervalType: 'yearly',
-			displayJetpackPlans: true
+			displayJetpackPlans: false
```

Comment:
Ok... I can agree that even if it involves more boilerplate in the initialization, that logic probably belong outside of the component ( in fact... wouldn't it be better if instead of injecting this `displayJetpackPlans` flag we injected the plans list? the `getPlansFeatures` method in https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/plans-features-main/index.jsx#L33 sound exactly that something that should be consuming a selector returning the proper plans for that site). 

Well, anyway, design discussion apart, if we keep this change, I really think we need to remove that default. Probably none of us is going to overlook that prop anytime soon, but I'm picturing a scenario where someone new to plans set up a new view where users can access to a plans-features-main (I don't know...a new NUX flow? a marketing landing page?) and they don't realize that they need to add the isJetpackButNotAT logic to the instantiation of the component... Let's force everyone to be explicit here and avoid any risk :)